src/components/Statistic.tsx

```diff
@@ -0,0 +1,40 @@
+import React, { useMemo } from "react";
+
+interface StatisticProps {
+	label: string;
+	value: number;
+	currency?: string;
+	isPercentage?: boolean;
+}
+
+export const Statistic = ({ label, value, currency, isPercentage = false }: StatisticProps) => {
+	const formatter = useMemo(
+		() =>
+			isPercentage
+				? Intl.NumberFormat("en", {
+						style: "percent",
```

Comment:
maybe the `isPercentage` can go into the `style` because the other values are the same? 
```	
style: isPercentage ? "percent" : "compact"
```

Reply:
That won't work, the percentage formatter uses `style` while the number formatter uses `notation`

Reply:
you can set the property to undefined if not applicable

Reply:
I prefer the first solution
maybe switching around the first undefined : "compact" but I'm not gonna argue about that if you don't like switching it.